sechub-pds-solutions/phan/composer.json

```
@@ -0,0 +1,5 @@
+{
+    "require": {
+        "phan/phan": "^5.4"
```

Use the exact version number. 

Furthermore, it would be nice to have this version number injected via variable (docker compose). You can achieve this by using a placeholder like "PHAN_VERSION" and replacing it with `sed` and the value from the variable.

```suggestion
        "phan/phan": "5.4"
```

---

Please have a look at my next commit about this improvement. I'm not sure if I resolved this problem correctly.